scripts/ci-install-deps.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+### Install all dependencies on Ubuntu 16.04 LTS (Xenial Xerus) for RChain dev environment
+
+
+## Set BASH environment so it will fail properly throwing exit code
+set -euxo pipefail
+
+## setup tmp directory
+tmp_dir="deps-tmp"
+if [ ! -d "${tmp_dir}" ]; then
+    mkdir ${tmp_dir}
+else
+    rm -rf ${tmp_dir}
+fi
+
+## Detect if running in docker container - setup using sudo accordingly
+if [[ $(cat /proc/self/cgroup  | grep docker) = *docker* ]]; then
+    echo "Running in docker container!"
+    sudo=""
+else
+	sudo="sudo"
+fi
+
+
+## Verify operating system (OS) version is Ubuntu 16.04 LTS (Xenial Xerus)
+# Add more OS versions as necessary. 
+version=$(cat /etc/*release | grep "^VERSION_ID" | awk -F= '{print $2}' | sed 's/"//g')
+if [[ "$version" == "16.04" ]]; then
+    echo "Running install on Ubuntu 16.04" 
+else
+    echo "Error: Not running on Ubuntu 16.04"
+    echo "Exiting"
+    exit
+fi
+
+## Resynchronize the package index files from their sources
+${sudo} apt-get update -yqq
+
+## Install g++ multilib for cross-compiling as rosette is currently only 32-bit 
+${sudo} apt-get install g++-multilib -yqq
+
+## Install misc tools 
+${sudo} apt-get install cmake curl git -yqq
+
+## Install needed crypto
+${sudo} apt-get install autoconf libtool -yqq
+cd crypto
+git clone https://github.com/bitcoin-core/secp256k1
+cd secp256k1
+./autogen.sh
+./configure --enable-jni --enable-experimental --enable-module-schnorr --enable-module-ecdh --prefix=$PWD/.tmp
```

Comment:
lose `--enable-module-schnorr` (it doesn't exist) and `--prefix=$PWD/.tmp` and in the following line, just `make` (no `install`)